### DISCUSSION TITLE
Instance Score

### DISCUSSION THREAD

**Discussion by msjindal** [talmolab/sleap#1682] [Help!]
Can I get the instance score for each body part and not just the overall instance score? Also is the instance score the accuracy?

**Comment by roomrys** [comment]
Hi @msjindal,

Only the instance score is displayed in the GUI and can be found in two places:
1. the "Labeling Suggestions" dock under the "Mean Score" column
2. the "Instances" dock under the "Score" column

The score for each body part is referred to as the "point score" and while it is not displayed in the GUI, it is stored for each `Node` in a `PredictedInstance`. Since point scores are not displayed in the GUI, you will need to use the API to access them through the `PredictedInstance.scores` property which will list all the point scores for each node in `PredictedInstance.nodes`.

From #1268, the point score is a
>non-zero likelihood that a pixel contains the center of a node given the annotated frames

The point score doesn't necessarily measure accuracy. A model can predict the location of a body part very accurately (w.r.t. where the user would have labeled the body part), but still lack confidence that the prediction is correct (due to perhaps a lack of examples it has seen of the body part in it's depicted state [think lighting, tilt, rotation, ANY variance in the appearance of a body part]). You can use point scores to determine which types of examples your model would like more annotations of (to increase its confidence when predicting) - even if these predictions are accurate, your model is struggling with some confidence issues.

Thanks,
Liezl

Related discussions:
- #1680
- #1268
